### ISSUE TITLE
Grid Menu Custom List of Commands are not showing

### ISSUE THREAD

**Issue by benitoparis** [ghiscoding/Angular-Slickgrid#244]
Hello Ghislain,

First thing first, i would like to thank you a lot for the job you have done with angular slickgrid.

I have an issue because we have just upgraded from angular 6 to angular 7 (so now we use "angular-slickgrid": "^2.9.6".

I have a button and i would like to show up the grid menu when the user click on it.
I have followed your example below:
https://github.com/ghiscoding/Angular-Slickgrid/blob/master/src/app/examples/grid-menu.component.ts

```ts
  toggleGridMenu(e) {
    if (this.angularGrid && this.angularGrid.extensionService) {
      const gridMenuInstance = this.angularGrid.extensionService.getSlickgridAddonInstance(ExtensionName.gridMenu);
      gridMenuInstance.showGridMenu(e);
    }
  }
```

The gridMenuInstance is found but the method gridMenuInstance.showGridMenu(e) doesn't seem to work. (the menu doesn't show up).

What shoud i use?

(with angular 6 and a former version of slickgrid i used the script below and it worked
```ts
// const gridMenuInstance = this.angularGrid.pluginService.gridMenuControl;
      //gridMenuInstance.showGridMenu(e);
```
Thank you for your help

Ben

**Comment by ghiscoding** [comment]
I released a new version last Friday `2,9,7` which fixes the Grid Menu from an external button in this [Example](https://ghiscoding.github.io/Angular-Slickgrid/#/gridmenu) and from the `gridMenuInstance`. The fix addresses the `showGridMenu` that stopped working at some point. I cannot pinpoint to the exact commit because GitHub seems to have website issue today, at least from my side. 

So just update to `2.9.7` and it should work.

**Comment by benitoparis** [comment]
Hello Ghislain,
I have just upgraded the library version. The menu is now showing up :-) but custom items i added to gridmenu options is not displayed on the menu. 

     customItems: [
          {
            iconCssClass: 'fa fa-filter text-danger',
            title: 'Restaurer la configuration par défaut',
            disabled: false,
            command: 'reset'
          }
        ],

Could you please have a look?
Again, thank you for the time you spend resolving issues.

Ben

**Comment by ghiscoding** [comment]
That seems to be another bug, might have been introduced when I change a lot of the code when I added unit testing of the services and extensions. 

> Could you please have a look?

Please remember that this is an open source project, available for free under the MIT license, and this library is developed and supported on a voluntary basis. I certainly would be happy to receive help from anyone willing to contribute.

**Comment by ghiscoding** [comment]
Now fixed and released under version `2.9.8`


Please up vote, if you like the lib, or disregard if you have already ⭐️ 
Thanks for reporting the bug

The [Example 9](https://ghiscoding.github.io/Angular-Slickgrid/#/gridmenu) now shows up the "Help" customItems as can be seen below.

![image](https://user-images.githubusercontent.com/643976/61827872-8d60b080-ae33-11e9-8e2c-26d930848dca.png)
